packages/lit-dev-content/site/tutorials/content/async-directive/08.md

```diff
@@ -0,0 +1,53 @@
+In this step, take a moment to confirm the `disconnect()` and `reconnect()`
+lifecycle are working as expected.
+
+Add some code to the example element that removes the host element from the DOM
+for a second, then puts it back in. In the `time-ago-example` element's
+template, add the `@click` event listener to the `<p>`:
+
+```ts
```

Comment:
HTML

Reply:
Hmm, so this is another tricky one we should be consistent about: when a Lit HTML snippet has TS `${expressions}` in it, formatting as `html` means the expressions don't get TS formatting and just look like strings. Formatting as `ts` means the HTML is formatted _different_ than the HTML formatter (but I think it still uses TSX formatting, maybe, which mostly still works). Do we have a preference here?

Reply:
maybe you can stay with TS and do:

```ts
return html`
  ...
  <p @click=${this.handleClick}>
    This page was rendered ${timeAgo(timeCreated)}.
  </p>
  ...
`;
```